src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseScanTests/LicenseExclusions.txt

```diff
@@ -77,9 +72,6 @@ src/efcore/test/EfCore.Tests/ChangeTracking/MemberEntryTest.cs|proprietary-licen
 #
 
 # False positive
```

Comment:
@ellahathaway - Based on your investigation on the remove of exclusions in this file, if some are legitimate can you check whether there are any instances where the comment also needs to be deleted (like this one) and update the PR accordingly?